platforms/codesandbox/package.json

```diff
@@ -4,21 +4,21 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "prestart": "yarn prisma db push --preview-feature --accept-data-loss; yarn prisma generate;",
+    "prestart": "npx prisma db push --preview-feature --accept-data-loss; npx prisma generate;",
     "start": "node src/index.js",
-    "deploy": "yarn ts-node deploy.ts"
+    "deploy": "npx ts-node deploy.ts"
   },
   "dependencies": {
     "@prisma/client": "4.13.0-dev.19",
     "express": "^4.17.1",
-    "node-fetch": "^2.6.0",
-    "puppeteer": "^19.0.0"
+    "node-fetch": "^2.6.0"
   },
   "devDependencies": {
     "@types/node": "14.18.36",
     "@types/node-fetch": "2.6.2",
     "@types/puppeteer": "5.4.7",
     "prisma": "4.13.0-dev.19",
+    "puppeteer": "^19.0.0",
```

Comment:
do we still need this at all?

Reply:
I think so since we use the new sandboxes, I'll give it a try. As I understood, pupeteer was just used to wake up the sandbox.